### PR TITLE
feat: add strimzi docs

### PR DESCRIPTION
**Pull request by jaikanthjay46**

### CODE REVIEW

File: data/docs/messaging-queues/kafka.mdx [hunk 1]
```diff
@@ -415,9 +415,417 @@ Run the collector with the above configuration :
...
+    matchLabels:
+      app: kafka-producer1
+  template:
+    metadata:
+      labels:
+        app: kafka-producer1
+    spec:
+      containers:
+      - name: producer
+        image: jaikanthjay46/kafka-consumer-test:latest
```
Comment (Calm-Rock) [inline data/docs/messaging-queues/kafka.mdx:749]: @jaikanthjay46 can you please put a note mentioning that the user should change the image to their own docker image.

File: data/docs/messaging-queues/kafka.mdx [hunk 2]
```diff
@@ -415,9 +415,417 @@ Run the collector with the above configuration :
...
+    matchLabels:
+      app: kafka-consumer1
+  template:
+    metadata:
+      labels:
+        app: kafka-consumer1
+    spec:
+      containers:
+      - name: consumer
+        image: jaikanthjay46/kafka-consumer-test:latest
```
Comment (Calm-Rock) [inline data/docs/messaging-queues/kafka.mdx:791]: Same as above for producer. Please add a note mentioning explicitly that the user has to change this with their own docker image.

File: data/docs/messaging-queues/kafka.mdx [hunk 3]
```diff
@@ -415,9 +415,417 @@ Run the collector with the above configuration :
...
+
+Apply the deployments:
+```bash
+kubectl apply -f kafka-applications.yaml
+```
+Please take note of otel service name `my-release-k8s-infra-otel-agent` in the above yaml file, you will need to replace it with your otel service name.
+It's of the format `<helm-release-name>-k8s-infra-otel-agent`.
+
+Continue to visualize your data in SigNoz as described in the previous sections.
+
```
Comment (Calm-Rock) [inline data/docs/messaging-queues/kafka.mdx:823]: Can you add an image of the final output in the messaging queue section that you see for Strimzi in this tab itself ?

**Review by Calm-Rock — CHANGES_REQUESTED** [review]

**Comment by shivanshuraj1333** [comment]
Thanks would review it when I get a breather.
In the mean time please take a look at Chitransh's comment.

File: data/docs/messaging-queues/kafka.mdx (hunk 3 above)
Comment (jaikanthjay46) [reply data/docs/messaging-queues/kafka.mdx:823]: The article already has them, will add them when we separate it into different docs.

Comment (Calm-Rock) [reply data/docs/messaging-queues/kafka.mdx:823]: Okay.

**Comment by Calm-Rock** [comment]
Resolved in https://github.com/SigNoz/signoz-web/pull/958

**Pull request closed without merging**